### PR TITLE
Add #content skiplink

### PR DESCRIPTION
**Pull request by Demotive**
screen shot 2016-06-03 at 09 49 07

### CODE REVIEW

File: app/views/elements.nunjucks [hunk 1]
```diff
@@ -1,76 +1,78 @@
 {% extends 'layout.nunjucks' %}
 
 {% block content %}
-  <h1>Heading one</h1>
-  <h2>Heading two</h2>
-  <h3>Heading three</h3>
-  <h4>Heading four</h4>
-  <h5>Heading five</h5>
-  <h6>Heading six</h6>
+  <main id="content" role="main" tabindex="-1">
```
Comment (teneightfive) [inline app/views/elements.nunjucks]: Should we move this to the main layout view? Seems to be shared across all templates at the moment.

File: assets/stylesheets/components/_skiplinks.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  .skiplinks--link {
+    color: $white;
+    display: inline-block;
+    left: -999em;
+    padding: ($baseline-grid-unit * 2);
+    position: absolute;
+
+    &:focus {
+      background-color: tint($blue, 95%);
+      border: 2px solid $yellow;
```
Comment (teneightfive) [inline assets/stylesheets/components/_skiplinks.scss]: Can we keep the focus outline/border colour consistent. We're using 3px for outline but only 2px here

File: app/views/elements.nunjucks (hunk 1 above)
Comment (Demotive) [inline app/views/elements.nunjucks]: I felt like keeping it flexible might be better in the long run? But happy to stick it in layout until we do need that flexibility?

File: assets/stylesheets/components/_skiplinks.scss (hunk 2 above)
Comment (Demotive) [inline assets/stylesheets/components/_skiplinks.scss]: :trollface:

File: app/views/elements.nunjucks (hunk 1 above)
Comment (teneightfive) [inline app/views/elements.nunjucks]: It's more just that it's a common part of the layout now that the skiplinks have been added. So they're effectively tied together. 

If we need to add it to every view it makes it easier to forget and miss out which means the skiplinks in the main layout then become broken.

Comment (Demotive) [inline app/views/elements.nunjucks]: Aye, fair enough - will shift it now :)

**Pull request merged**